compile/packages/index.js

```diff
@@ -24,7 +24,7 @@ class OpenWhiskCompilePackages {
   }
 
   renameManifestPackages() {
-    if (!this.serverless.service.resources.packages) return;
+    if (!this.serverless.service.resources) return;
```

Comment:
What happens if they have a `service.resources` section but not a `service.resources.packages` section? I think it'll end up with Object.keys(null or undefined) which'll throw a type error?